src/components/home/partners/PrimaryPartner.jsx

```diff
@@ -1,8 +1,83 @@
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+
+const sponsorValues = [
+  {
+    title: 'Advocacy',
+    description:
+      "We amplify your voice and bridge industry and government to drive policies that ensure the growth and success of the region's tech industry."
+  },
+  {
+    title: 'Community',
+    description:
+      'We bring the regional tech community together through in-person and online events to deepen relationships and build meaningful connections.'
+  },
+  {
+    title: 'Visibility',
+    description:
+      'We shine a spotlight on the companies and people that power the region’s innovation economy and leverage our platform to promote the local tech sector.'
+  },
+  {
+    title: 'Resources',
+    description:
+      'We connect our members to resources to help their businesses grow and succeed through public and private sector partnerships.'
+  }
+];
+
+const displaySponsorValuesInGrid = (value, index) => {
+  return (
+    <Grid value xs={6} key={index}>
+      <Typography variant="body1" p={'0 10% 18% 15%'}>
+        <Typography variant="body1" pb={'6%'}>
```

Comment:
Title should be more bold (and probably a heading of some sort) ot the paragraph below it.

Reply:
changed in next push. used `<strong> ` because allegedly it's better for screen readers than mui font weight.